docs/components/stepper/index.html

```diff
@@ -32,7 +30,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Account</span>
```

Comment:
Please update all of these `beta` classes.

Reply:
Changes have been done, please review.